collection-scripts/gather_metrics

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Should this be added to the `gather` script so that metrics (or some specification of default metrics) are collected by default?

Reply:
I asked the same questions in https://github.com/openshift/must-gather/pull/389/files#r1658786615 to use this to collect some metrics out of the box in `gather_monitoring`, but I can't think of any specific metrics right now, we can gradually build this list as we use `gather_metrics`.
(we can also run many `dumps` with different time ranges by default, if we also need to dump some high cardinality metrics by default)

Reply:
One idea could be to collect the telemetry metrics (e.g. id would be useful for disconnected clusters). Not required for this PR though.

Reply:
> One idea could be to collect the telemetry metrics (e.g. id would be useful for disconnected clusters). Not required for this PR though.

That can be a good start indeed, I was worried regarding the effort required to make sure the lists stay synced. but I think we can make the script fetch the list from the `openshift-monitoring/telemetry-config` configmap on the cluster.

Even though we try to control the cardinalities while adding those metrics to that configmap, I’ll still need to run some tests to make sure the operation will go easy on Prometheus and won’t result in huge dumps.

Reply:
perhaps something similar is already included in those telemetry metrics - something I regularly ask for is alert history for the last 24hrs: minus the watchdog (just to make it easier to read) the must-gather only gives me whats currenlty firing  

`query=sum by (alertname, severity)(ALERTS{alertname!='Watchdog'})" --data-urlencode "&start=2024-07-09T00:00:00.781Z" --data-urlencode "end=2024-07-10T00:00:00.781Z" --data-urlencode "step=5m"  'http://localhost:9090/api/v1/query_range?`